### PR TITLE
Use a distinct `guiPort`

### PR DESCRIPTION
**Pull request by geoknee**
Following today's standup discussion, this was felt to be a reasonable compromise. One more port, but quite a lot of code saved and no need to use a global variable. 

Stacked on #1459

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -113,6 +114,13 @@ func main() {
 			Category:    "Connectivity:",
 			Destination: &rpcPort,
 		}),
+		altsrc.NewIntFlag(&cli.IntFlag{
+			Name:        GUI_PORT,
+			Usage:       "Specifies the tcp port for the Nitro Connect GUI.",
+			Value:       5005,
+			Category:    "Connectivity:",
```
Comment (NiloCK) [inline main.go:121]: Minor side issue:

Do these categories ever get used?
- yes: extract into constants
- no: maybe scrub, replace with empty string or `"categoriesnotused"` const

File: main.go [hunk 2]
```diff
@@ -141,14 +149,7 @@ func main() {
 				return err
 			}
 
-			uiPort := uint(rpcPort)
-			// If we're using nats we can't re-use the RPC port for the UI, so we choose another
```
Comment (NiloCK) [inline main.go:145]: definitely good to get rid of port juggling logic as a function of other config options.

**Review by NiloCK — APPROVED** [review]
:+1:

File: main.go (hunk 1 above)
Comment (geoknee) [reply main.go:121]: They do get used in the "help" function I believe, to organise the output. Good call on constants.

**Pull request merged**